junit/src/main/java/org/kaazing/k3po/junit/rules/SpecificationStatement.java

```diff
@@ -55,10 +55,10 @@ public void evaluate() throws Throwable {
             // start the script execution
             new Thread(scriptFuture).start();
 
-            // wait for script to be prepared (all binds ready for incoming connections from statement)
-            latch.awaitPrepared();
-
             try {
+                // wait for script to be prepared (all binds ready for incoming connections from statement)
```

Comment:
About this one I'm not sure, it complicates the error handling. I'm trying an option to have a separate try-catch for awaitPrepared. When I'll get something working fine I'll commit it.